### PR TITLE
More cleanup

### PR DESCRIPTION
**Pull request by damacus**
### Description

[Describe what this change achieves]

### Issues Resolved

[List any existing issues this PR resolves]

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

### CODE REVIEW

File: resources/pg_gem.rb [hunk 1]
```diff
@@ -1,7 +1,8 @@
-# frozen_string_literal: false
+# frozen_string_literal: true
+# encoding: utf-8
```
Comment (tas50) [inline resources/pg_gem.rb]: Do we need to define the encoding here?

File: resources/pg_gem.rb [hunk 2]
```diff
@@ -47,76 +58,39 @@
...
-      lib_builder.run_action(:run)
-
-      lib_maker = execute 'make pg gem lib' do
-        command 'make'
-        cwd ext_dir
-        action :nothing
-      end
-      lib_maker.run_action(:run)
+  if ruby_version < 2.0
+    Chef::Log.fatal("pg gem requires a system Ruby installation of 2.0 or higher. \n Please install a global Ruby")
```
Comment (tas50) [inline resources/pg_gem.rb]: You want to raise here so you actually fail the run

File: resources/pg_gem.rb [hunk 3]
```diff
@@ -47,76 +58,39 @@
...
+  gem_package 'pg' do
+    clear_sources new_resource.clear_sources if new_resource.clear_sources
+    include_default_source new_resource.include_default_source if new_resource.include_default_source
+    gem_binary new_resource.gem_binary if new_resource.gem_binary
+    options new_resource.options if new_resource.options
+    source new_resource.source if new_resource.source
+    timeout new_resource.timeout if new_resource.timeout
+    version new_resource.version if new_resource.version
+    action :install
+    not_if { ruby_version < 2.0 }
```
Comment (tas50) [inline resources/pg_gem.rb]: We shouldn't need to double guard against old ruby

File: resources/pg_gem.rb (hunk 1 above)
Comment (damacus) [reply resources/pg_gem.rb]: Cookstyle probably.

File: resources/pg_gem.rb (hunk 2 above)
Comment (tas50) [reply resources/pg_gem.rb]: I meant you should just raise "stuff be wrong!" instead of the chef log.

File: resources/pg_gem.rb [hunk 4]
```diff
@@ -47,76 +58,36 @@
...
-      end
-      lib_builder.run_action(:run)
-
-      lib_maker = execute 'make pg gem lib' do
-        command 'make'
-        cwd ext_dir
-        action :nothing
-      end
-      lib_maker.run_action(:run)
+  raise ArgumentError, "pg gem requires a system Ruby installation of 2.0 or higher." if uby_version < 2.0
```
Comment (duncaan) [inline resources/pg_gem.rb]: Did you mean `ruby_version`?

**Comment by damacus** [comment]
Most of ruby_gem_pg is a mess right now.

There's a few discussions about why. So I'm not surprised if there are some
bugs in here.

We/i really want this to work on all platforms or none.

**Pull request merged**